mica-web-model/src/main/protobuf/Mica.proto

```diff
@@ -492,6 +504,7 @@ message HarmonizationDatasetDto {
   optional string networkId = 4;
   repeated DatasetDto.NetworkTableDto networkTables = 5;
   optional HarmonizationStudyReferenceDto studyReference = 6;
+  repeated DatasetDto.HarmonizationTableDto harmonizationTables = 7;
   extensions 1000 to max;
 
   message HarmonizationStudyReferenceDto {
```

Comment:
The HarmonizationStudyReferenceDto and the project + table should have been merged into a HarmonizationTableDto, just like done for the StudyDataset + StudyTable. UI is to be updated accordingly.